docs/src/main/sphinx/connector/influxdb.rst

```diff
@@ -0,0 +1,164 @@
+=======================
+InfluxDB connector
+=======================
+
+The InfluxDB Connector allows access to `InfluxDB <https://www.influxdata.com/>`_ data from Trino.
+This document describes how to setup the InfluxDB Connector to run SQL queries against InfluxDB.
+
+Requirements
+------------
+
+To connect to InfluxDB, you need:
+
+* InfluxDB v1.8.0 or higher.(v2.x not currently supported)
```

Comment:
For my understanding, for 2.x do we need additional libraries - are they Open Sourced or ?

Reply:
> yes, we need additional libraries.  the libraries is open sourced.

Reply:
In this case why don't we directly support the latest version v2.something ?

Reply:
API for v1.x  and v2.x are not compatible, influxdb v1.x is open sourced and used more widely now as i know.  So you means we do both v1.x  and v2.x connector in this PR? or v2.x supported in next pr i think. cause that's maybe many works to do.

Reply:
>  So you means we do both v1.x and v2.x connector in this PR

No... If 2.x was widely adopted then we could directly implement for 2.x, since v1.x is widely used we could consider it.